### PR TITLE
Issue 13 implementation

### LINKED ISSUE #13: Configuration.json mandatory
It would be great if the configuration.json was not mandatory.

- Git support is not always required, if only checking local files
- Database could have a default local value (ex: "..\nugetvisualizer.db" )

This could improve the deployment speed and still keep the configuration available for some more advanced cases :)

### PR DESCRIPTION
**Pull request by organom**
If no configuration.json is available or the settings inside the file is not available, then the default values are used.
fixes #13

### CODE REVIEW

**Comment by sepharg** [comment]
@organom it looks like the build has failed with some tests, could you fix that please before I merge it in?

File: NugetVisualizer/Core/ConfigurationHelper.cs [hunk 1]
```diff
@@ -17,7 +17,7 @@ private IConfigurationSection GetSection(string sectionName)
         }
 
         public bool UseSqlLite => true;
-        public string Dbpath => GetSection("Dbpath").Exists() ? GetSection("Dbpath").Value : "nugetvisualizer.db";
+        public string Dbpath => GetSection("Dbpath").Exists() ? GetSection("Dbpath").Value : "..\\nugetvisualizer.db";
```
Comment (sepharg) [inline NugetVisualizer/Core/ConfigurationHelper.cs]: this is not platform agnostic, will fail on linux

Comment (organom) [reply NugetVisualizer/Core/ConfigurationHelper.cs]: will take care of this :)

**Comment by sepharg** [comment]
@organom where you able to fix the tests?

**Comment by organom** [comment]
Still didn't had time to look into this :( 
unfortunately with the latest code changes its not possible to merge anymore. canceling the pull request and making a new one in the future

**Pull request closed without merging**